src/main/java/com/speaker/service/AccountConverter.java

```diff
@@ -0,0 +1,27 @@
+package com.speaker.service;
+
+import com.speaker.DTO.AccountDTO;
+import com.speaker.entities.Account;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class AccountConverter {
+    public List<AccountDTO> convertToAccountDTO(List<Account> account) {
```

Comment:
let the method take an object, do stream in service

Reply:
because we possibly will be using this method in other cases where the input parameter will be the object instead of a list